node/src/Pos/Wallet/Web/Secret.hs

```diff
@@ -34,7 +35,7 @@ instance Buildable WalletUserSecret where
     build WalletUserSecret{..} =
         bprint ("{ root = "%addressF%", set name = "%build%
                 ", wallets = "%pairsF%", accounts = "%pairsF%" }")
-        (makePubKeyAddress $ encToPublic _wusRootKey)
+        (makePubKeyAddress undefined $ encToPublic _wusRootKey)
```

Comment:
Well, if there is no difference, I would rather use bootstrap era distribution, because it takes less space. Regarding helper: got it, will look into it.